### ISSUE TITLE
the default for PackageProcessor.ServerUrl should be set to port 80

### ISSUE THREAD

**Issue by FiveHundred** [SymbolSource/SymbolSource#4]
In SymbolSource.Processor.Console/app.config, we see this line:

`<add key="PackageProcessor.ServerUrl" value="http://localhost:57738" />`

I think it should be set to something like this:

`<add key="PackageProcessor.ServerUrl" value="http://localhost" />`

The reason is that there's very spare documentation on how source servers work, and because of this seeing that port as the default is confusing. Confusion around this setting resulted in an issue that took me quite a while to resolve, but maybe I'm just not particularly bright.

**Comment by TripleEmcoder** [comment]
To my defense, this is mentioned in the README:

> `PackageProcessor.ServerUrl` - the URL, including protocol, under which Server is being exposed to symbol consumers, embedded in PDB files during indexing

However it it could be a bit lost among other options.

Dropping the port doesn't really solve the issue, as the hostname also needs to be set correctly - clients need to reach the symbol server using the `ServerUrl` which gets embedded in PDB files.

I think that an empty string plus a check on startup would be the best default, as the processor would then fail gracefully with an explanation of the reason. Would you have a some spare time to do a pull request?

**Comment by mrmbernardi** [comment]
That's all true, but the port is still significant. In the absence of source server documentation I had assumed that symbols package uploads and PDB downloads by default occured through port 80, while source downloads by default occured through port 57738. That's what this value suggests. 

Of course, when you actually run it on a default IIS configuration by default it's expecting source downloads on port 80.
